lang/python/python-package.mk

```diff
@@ -21,6 +21,10 @@ PYTHON:=python$(PYTHON_VERSION)
 
 PYTHONPATH:=$(PYTHON_LIB_DIR):$(STAGING_DIR)/$(PYTHON_PKG_DIR):$(PKG_INSTALL_DIR)/$(PYTHON_PKG_DIR)
 
+ifeq ($(strip $(PYTHON_PKG_OLD)),)
```

Comment:
this patch is big;
and touches on plenty of sensitive stuff ;
i think this may also have to be split into smaller chunks and be it's own PR ;

generally, i would not use `PYTHON_PKG_OLD` ; i don't think we need to be backwards compatible for building stuff ; this could limit people from copying packages from master into their own release branches, but i don't feel we should care that much for those cases [mostly because it would generate more questions & problems than it's worth]

Reply:
The reason for PYTHON_PKG_OLD was signifcant number of packages broke with this commit (due to a lot of custom stuff that this makes unnecessary) and I wanted to decouple updating the packages from the build system change.  Definitely will split into a separate PR later today.  I'm also going to review the package breakage and show you some examples for your thoughts on best option going forward (in the new PR).